spec/models/availability_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: availabilities
+#
+#  id            :integer          not null, primary key
+#  user_id       :integer
+#  day           :integer
+#  time_range_id :integer
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#  status        :integer
+#
+# Indexes
+#
+#  index_availabilities_on_time_range_id  (time_range_id)
+#  index_availabilities_on_user_id        (user_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (time_range_id => time_ranges.id)
+#  fk_rails_...  (user_id => users.id)
+#
+
+require 'rails_helper'
+
+RSpec.describe Availability, type: :model do
+  it { should belong_to(:user) }
+  it { should belong_to(:time_range) }
+  it 'saves given a valid Availability' do
+    create(:time_range)
+    create(:user)
+    availability = Availability.new(user_id: User.take.id, time_range_id: TimeRange.take.id, status: 1, day: 6)
```

Comment:
Use factories. Find the `factory_bot` getting started guide, or check out others' tests

Reply:
Oh actually you have used it :o

You can do something like `create(:availability, status: 1)` if you wish to override it